db/migrate/20100929044155_add_persistence_token_to_users_and_admins.rb

```diff
@@ -1,13 +0,0 @@
-class AddPersistenceTokenToUsersAndAdmins < ActiveRecord::Migration
```

Comment:
You seem to have scooped up a lot of database cleanup which isn't related to the locale preference and should be a separate pull request. It's not the end of the world if all these migration files need to be deleted anyway, but it's a good practice not to mix concerns when submitting a pull request by ensuring only changes that are relevant to the change at hand are committed on that branch.